projects/igniteui-angular/src/lib/grids/hierarchical-grid/row-island.component.ts

```diff
@@ -184,7 +184,16 @@ export class IgxRowIslandComponent extends IgxHierarchicalGridBaseDirective
     protected override actionStripComponents: QueryList<IgxActionStripToken>;
 
     /**
-     * @hidden
+     * Event emitted when a layout of the row island is changed.
+     * ```html
+     * <igx-hierarchical-grid [data]="Data" [autoGenerate]="true">
+     *      <igx-row-island [key]="'childData'" (layoutChange)="layoutChange($event)" #rowIsland>
+     *          <!-- ... -->
+     *      </igx-row-island>
+     * </igx-hierarchical-grid>
+     * ```
+     *
+     * @memberof IgxRowIslandComponent
      */
     @Output()
     public layoutChange = new EventEmitter<any>();
```

Comment:
Um, I don't see a matching property, so not sure what this event is about. Internal?

Reply:
I guess so, reverting is as well